cmd/influxd/backup_util/internal/data.pb.go

```diff
@@ -30,18 +18,39 @@ var _ = math.Inf
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.GoGoProtoPackageIsVersion2 // please upgrade the proto package
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
```

Comment:
I thought we were stuck on protobuf version 2?

Reply:
I was also confused to see this, but many of the `.proto`s in the code already have `syntax = "proto3"`: https://github.com/influxdata/influxdb/blob/master-1.x/storage/reads/datatypes/storage_common.proto#L1

Reply:
From Slack discussion: this field is unrelated to the `syntax` of individual `.proto` files. Instead, it's a marker for the version of `gogo/protobuf` used to generate the `.pb.go` file. The markers are generated to catch mismatches when a dependency contains a `.pb.go` generated with a newer version of `gogo/protobuf` that the main module doesn't support.

For example, if enterprise still depended on `v1.1.1` of gogo, and we merged this change, the next PR to update influxdb would fail to compile on this line because the `GoGoProtoPackageIsVersion3` constant wouldn't be present.